### PR TITLE
Fix eval in regression test

### PR DESCRIPTION
**Pull request by ebsmothers**
The config name and the results parsing in our regression test job are incorrect.

This is actually a bit awkward to test now that we (a) don't allow creating PRs from a fork, and (b) don't let forks access the S3 bucket containing regression test artifacts.

So for now I've tested it locally, which I guess is better than nothing?

```
pytest tests/regression_tests/test_llama2_7b.py -m slow_integration_test
...
====== 1 passed, 1 warning in 267.94s (0:04:27) ======
```

### CODE REVIEW

File: tests/regression_tests/test_llama2_7b.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def test_finetune_and_eval(self, tmpdir, caplog, monkeypatch):
         runpy.run_path(TUNE_PATH, run_name="__main__")
         eval_cmd = f"""
         tune run eleuther_eval \
-            --config eleuther_eval \
+            --config eleuther_evaluation \
```
Comment (SalmanMohammadi) [inline tests/regression_tests/test_llama2_7b.py:53]: How was this test working before??

**Comment by SalmanMohammadi** [comment]
> This is actually a bit awkward to test now that we (a) don't allow creating PRs from a fork, and (b) don't let forks access the S3 bucket containing regression test artifacts.

I missed this, but we're no longer running integration tests in our CI?

File: tests/regression_tests/test_llama2_7b.py (hunk 1 above)
Comment (ebsmothers) [reply tests/regression_tests/test_llama2_7b.py:53]: It wasn’t, it’s been failing in CI for a while now. But it doesn’t run on PRs or anything, only nightly

**Comment by ebsmothers** [comment]
> > This is actually a bit awkward to test now that we (a) don't allow creating PRs from a fork, and (b) don't let forks access the S3 bucket containing regression test artifacts.
> 
> I missed this, but we're no longer running integration tests in our CI?

So we actually have two types of integration tests: recipe tests and regression tests. Recipe tests always run on PRs but only use small checkpoints. Regression tests use the full-size model and run nightly. Currently this is the only regression test we have, but we’ve been wanting to add more and just haven’t had time (e.g. it’d be nice if we could test memory or perf of some of our models too).

**Review by SalmanMohammadi — APPROVED** [review]

**Comment by SalmanMohammadi** [comment]
I'll raise an issue : )

**Pull request merged**